dask/dataframe/tests/test_optimize_dataframe.py

```diff
@@ -15,10 +15,8 @@
 
 
 def test_column_optimizations_with_bcolz_and_rewrite():
-    try:
-        import bcolz
-    except ImportError:
-        return
+    pytest.importorskip('bcolz')
```

Comment:
No worries. And yes, those failures seem to be due to the new botocore release, and are unrelated. Thanks for the fix!